akka-http2-support/src/main/scala/akka/http/impl/engine/http2/FrameLogger.scala

```diff
@@ -76,8 +76,11 @@ private[http2] object FrameLogger {
           }.mkString(", ")
           LogEntry(0, "SETT", settingsInfo)
 
-        case SettingsAckFrame ⇒
-          LogEntry(0, "SETA", "")
+        case SettingsAckFrame(s) ⇒
+          val acksInfo = s.map {
```

Comment:
Can you create a method to be reused with the above? In case we want to change the format later on.